third_party/mesa3d/src/util/rust/defines.rs

```diff
@@ -25,6 +25,7 @@ pub const MESA_HANDLE_TYPE_MEM_DMABUF: u32 = 0x0002;
 pub const MESA_HANDLE_TYPE_MEM_OPAQUE_WIN32: u32 = 0x0003;
 pub const MESA_HANDLE_TYPE_MEM_SHM: u32 = 0x0004;
 pub const MESA_HANDLE_TYPE_MEM_ZIRCON: u32 = 0x0005;
+pub const MESA_HANDLE_TYPE_MEM_AHB: u32 = 0x0006;
```

Comment:
https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/38973

should i merge that first, and then sync?

Reply:
Looks much better, thanks!